tools/simple-httpbin/src/main/java/org/enso/shttp/SimpleHTTPBin.java

```diff
@@ -0,0 +1,97 @@
+package org.enso.shttp;
+
+import com.sun.net.httpserver.HttpHandler;
+import com.sun.net.httpserver.HttpServer;
+import sun.misc.Signal;
+
+import java.io.IOException;
+import java.net.InetSocketAddress;
+
+public class SimpleHTTPBin {
+
+  private final HttpServer server;
+  private final State state;
+
+  public SimpleHTTPBin(String hostname, int port) throws IOException {
+    InetSocketAddress address = new InetSocketAddress(hostname, port);
+    server = HttpServer.create(address, 0);
+    server.setExecutor(null);
+    state = new State();
+  }
+
+  public void start() {
+    server.start();
+    state.start();
+
+    try {
+      while (state.isRunning()) {
+        Thread.sleep(1000);
+      }
+    } catch (InterruptedException e) {
+      e.printStackTrace();
+    } finally {
+      server.stop(0);
+    }
+  }
+
+  public void stop() {
+    state.stop();
+  }
+
+  public void addHandler(String path, HttpHandler handler) {
+    server.createContext(path, handler);
+  }
+
+  public static void main(String[] args) {
+    if (args.length != 2) {
+      System.err.println("Usage: SimpleHTTPServer <host> <port>");
```

Comment:
```suggestion
      System.err.println("Usage: SimpleHTTPBin <host> <port>");
```

Seems this is inconsistent with the class name?